x11-wm/i3/i3-4.14.1.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="https://i3wm.org/downloads/${P}.tar.bz2"
 
```

Comment:
Per our policy, package name should go before the "Revert ..." part, i.e. sth like:

    x11-wm/i3: Revert "add ~arm64, ..."

I'll fix it for you and merge.